pkg/apis/extensions/types.go

```diff
@@ -638,3 +638,39 @@ type ClusterAutoscalerList struct {
 
 	Items []ClusterAutoscaler `json:"items"`
 }
+
+// A pod selector is a label query over a set of pods. The result of matchLabels and
+// matchExpressions are ANDed. An empty podSelector matches all objects. A null
+// podSelector matches no objects.
+type PodSelector struct {
+	// matchLabels is a map of key value pairs. A single key value in the matchLabels
+	// map is equivalent to a podSelectorRequirement where podSelectorRequirement.key
```

Comment:
If you want to introduce the "pod selector requirement" terminology here, as well, just use english rather than pseudo-fields.